apps/mobile/src/slices/auth/actions.ts

```diff
@@ -25,6 +29,48 @@ const getAuthenticatedUser = createAsyncThunk<
 	return await authApi.getAuthenticatedUser();
 });
 
+const requestResetPassword = createAsyncThunk<
+	boolean,
+	EmailDto,
+	AsyncThunkConfig
+>(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
+	const { authApi, toastMessage } = extra;
+
+	const isSuccessful = await authApi.requestResetPassword(emailPayload);
+
+	if (isSuccessful) {
+		toastMessage.success({ message: ToastMessageContent.LINK_SENT });
+	}
+
+	return isSuccessful;
+});
+
+const resetPassword = createAsyncThunk<
+	boolean,
+	ResetPasswordDto,
+	AsyncThunkConfig
+>(`${sliceName}/reset-password`, async (emailPayload, { extra }) => {
+	const { authApi, toastMessage } = extra;
+
+	const isSuccessful = await authApi.resetPassword(emailPayload);
+
+	if (isSuccessful) {
+		toastMessage.success({ message: ToastMessageContent.PASSWORD_UPDATED });
```

Comment:
Don't we need to show the toast if the request failed?

Reply:
It is shown on error, we have error notification implemented in our middleware.